### PR TITLE
Change PWA theme-color for frontend to adhere to system theme

### LINKED ISSUE #1338: chromium pwa white title bar does not respect dark system theme
image
all in the title

### PR DESCRIPTION
**Pull request by smarizvi110**
Resolves #1338

### CODE REVIEW

**Comment by Keavon** [comment]
Your PR changes the indentation to tabs in one, but not both, files. Could you please either make them both tabs or keep them as both spaces?

**Comment by smarizvi110** [comment]
> Your PR changes the indentation to tabs in one, but not both, files. Could you please either make them both tabs or keep them as both spaces?

Thanks for the heads up, done.

**Comment by Keavon** [comment]
Except now you replaced all the indentation in index.html...

**Comment by smarizvi110** [comment]
> Except now you replaced all the indentation in index.html...

Ah sorry. Let me just fix that. I must've accidentally configured the indentations to be more like rust's way for that too. Fixed now. Thank you for the patience 🙏

File: frontend/index.html [hunk 1]
```diff
@@ -13,12 +13,12 @@
...
 	<meta name="msapplication-TileColor" content="#ffffff">
-	<meta name="theme-color" content="#ffffff">
+	<meta name="theme-color" content="auto">
 	<meta name="color-scheme" content="dark only">
 	<meta name="darkreader-lock">
 	<!-- INDEX_HTML_HEAD_REPLACEMENT -->
-</head>
-<body>
+	</head>
+	<body>
```
Comment (Keavon) [inline frontend/index.html:21]:
```suggestion
</head>
<body>
```

File: website/static/site.webmanifest [hunk 2]
```diff
@@ -13,7 +13,7 @@
             "type": "image/png"
         }
     ],
-    "theme_color": "#ffffff",
+    "theme_color": "auto",
```
Comment (Keavon) [inline website/static/site.webmanifest:16]:
```suggestion
    "theme_color": "#ffffff",
```

**Comment by Keavon** [comment]
Where did you find that `"auto"` as a value is a solution to #1338? I can't find any reference to that value in my research, and it looks like [this issue](https://github.com/w3c/manifest/issues/975) that would automatically adapt to light and dark themes is still an open discussion that, I don't think, has been implemented into browsers. I'm concerned this isn't a real solution to the issue (and for that reason I'm going to temporarily close this but please reply and I'm happy to reopen it). Thanks.

**Comment by smarizvi110** [comment]
> Where did you find that `"auto"` as a value is a solution to #1338? I can't find any reference to that value in my research, and it looks like [this issue](https://github.com/w3c/manifest/issues/975) that would automatically adapt to light and dark themes is still an open discussion that, I don't think, has been implemented into browsers. I'm concerned this isn't a real solution to the issue (and for that reason I'm going to temporarily close this but please reply and I'm happy to reopen it). Thanks.

Stumbled across this as a fix that could work for PWAs by just trying it. It seemed to work on my end, and the PWA would change the title bar color in accordance with the theme. But yes, I agree that it doesn't seem to be a widely supported option and the issue you mentioned is definitely worth looking in to. I'll be on the lookout for a better solution with regards to the issue you mentioned, but this PR should indeed remain closed for now. Thanks!

**Pull request closed without merging**